<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>日期时间</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>时间</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="ledtDate"/>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="ledtTime"/>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="btnSetDate">
          <property name="text">
           <string>设置日期</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>日期时间</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QTimeEdit" name="timeEdit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="ledtDateTime">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QDateEdit" name="dateEdit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="btnSetTime">
          <property name="text">
           <string>设置时间</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>日期</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="btnSetDateTime">
          <property name="text">
           <string>设置时间日期</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnCurDateTime">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>当前日期和时间</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_4">
          <property name="contextMenuPolicy">
           <enum>Qt::NoContextMenu</enum>
          </property>
          <property name="toolTipDuration">
           <number>0</number>
          </property>
          <property name="text">
           <string>字符显示</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>定时器</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="btnClockStart">
              <property name="text">
               <string>开始</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClockEnd">
              <property name="text">
               <string>停止</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbTimeFlow">
              <property name="text">
               <string>流逝时间</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="lbClockPeriod">
              <property name="text">
               <string>定时周期</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spbClockPeriod"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnSetPeriod">
              <property name="text">
               <string>设置周期</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLCDNumber" name="lcdHour"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdMinute"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdSecond">
              <property name="segmentStyle">
               <enum>QLCDNumber::Filled</enum>
              </property>
              <property name="intValue" stdset="0">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>日历选择</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>选择日期: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ledtSelectDateTime"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>536</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
